main.js

```diff
@@ -0,0 +1,33 @@
+document.querySelector('.header').addEventListener('click', () => {
+  // Creating the X sumbol by changing the .bar div position.
+  document.querySelector('#bar1').classList.toggle('bar1-clicked');
+  document.querySelector('#bar2').classList.toggle('bar2-clicked');
+  document.querySelector('#bar3').classList.toggle('bar3-clicked');
+
+  // poping up the mobile menu
+  document.querySelector('.mobile-menu').classList.toggle('mobile-menu-trans');
+});
+
+// closing the popup menu onclicking the popup menu items
+
+const menu = Array.from(document.querySelectorAll('.menu a'));
+menu.forEach((link) => {
+  link.addEventListener('click', () => {
+    document.querySelector('.mobile-menu').classList.remove('mobile-menu-trans');
+    document.querySelector('#bar1').classList.remove('bar1-clicked');
+    document.querySelector('#bar2').classList.remove('bar2-clicked');
+    document.querySelector('#bar3').classList.remove('bar3-clicked');
+  });
+});
+
+// activating the dropdown menu
+const dropDown = document.querySelector('.dropdown-arrow');
+dropDown.addEventListener('click', () => {
+  // dispalying the hidden speakers
+  document.querySelector('.speaker-6').classList.toggle('dropdown-dispaly');
+  document.querySelector('.speaker-5').classList.toggle('dropdown-dispaly');
+  document.querySelector('.speaker-4').classList.toggle('dropdown-dispaly');
+
```

Comment:
- [ ] One of the core project requirements is to ensure that the section "Featured speakers" is generated dynamically with JavaScript. Kindly ensure you do not hardcode any HTML in the speakers section and use javascript to dynamically generate your speakers instead. You can use an array to store the objects
``` javascript
const speakerList = [ { name: "Speaker Name",... } ]

window.onload = () => {
  speakerList.map(speaker => {
    // code to build the new HTML element, it can use `createElement()` or just template strings(``).
    // it could attach the new element to a parent inside within the loop
    const speakerContainer = document.getElementById("element-id")
    speakerContainer.appendChild(dynamicElements)
    // OR
    speakerContainer.innerHTML = dynamicElements
    // If used inside a loop the innerHTML method should use `+=` instead of just `=`
    })
   // or accumulate the generate elements in a variable and attach them all after it
}

```
Kindly add this functionality.

![image](https://user-images.githubusercontent.com/90556221/183048364-5fc9b0ee-caa5-42a4-b5b8-32a337dc6cb3.png)

Reply:
Added the same.